frontend/app/src/next/components/runs/run-event-log/run-event-log.tsx

```diff
@@ -461,9 +460,11 @@ export function RunEventLog({ workflow, onTaskSelect }: RunEventLogProps) {
                         <p
                           className={cn(
                             'font-medium shrink-0',
+
                             RunStatusConfigs[
                               EVENT_CONFIG[event.eventType].status
-                            ]?.textColor || 'text-gray-500',
+                            ]?.primary || 'text-gray-500',
+                            'bg-transparent',
```

Comment:
I know we do this a lot - I think it's more Tailwind-y to use [data attribute variants](https://tailwindcss.com/blog/tailwindcss-v3-2#data-attribute-variants) for this sort of thing. It's not blocking or anything, but I think at some point we should do this